deps.edn

```diff
@@ -2,20 +2,20 @@
  :deps {org.clojure/clojure {:mvn/version "1.11.1"}
         com.cognitect.aws/api {:mvn/version "0.8.539"}
         meander/epsilon {:mvn/version "0.0.650"}
-        org.clojure/tools.logging {:mvn/version "1.2.1"}}
+        org.clojure/tools.logging {:mvn/version "1.2.4"}}
  :aliases
  {:test
-  {:main-opts ["-m" "kaocha.runner"]
-   :extra-paths ["test"]
+  ;; This profile doesn’t use :main-opts to start the test runner because that prevents some IDEs
+  ;; (such as Calva) from starting a REPL using this profile. So instead, we run the tests with the
+  ;; script bin/test (as recommended by the Kaocha docs).
```

Comment:
How does Calva do profile selection? Emacs has dir-locals, where it sets -C/-R, and hence brings in deps and resources without including main-opts or exec-fns.

Reply:
I don’t know.

All I know about this is that when I run Calva’s jack-in command and select the profile `:test` it shows me this warning dialog:

![image](https://user-images.githubusercontent.com/99206340/166333981-e094c69e-fe45-4488-94e0-9b95d215fd4a.png)

…and then the REPL does indeed not start successfully in such a way that Calva can connect to it.

I figured this change would be acceptable since (a) it’s easier to type `bin/test` than `clojure -M:test` and (b) this is the setup recommended by this project’s test runner, Kaocha.

Reply:
Yeah, this is fine. Though as a heads up we're going to have to change this in multiple locations. (And if it turns out we're doing that in multiple locations, I'm wondering if bb tasks aren't better suited than a shell script.)